index.html

```diff
@@ -211,5 +211,54 @@ <h1 class="about-heading">About me</h1>
         </li>
       </ul>
     </section>
+    <section class="contact-section">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form class="form-control">
+        <input
+          placeholder="Full Name"
+          class="text-field"
+          type="text"
+          minlength="30"
+          required
+        />
+        <input
+          placeholder="Email Address"
+          class="text-field"
+          type="email"
+          required
```

Comment:
- Please your webpage is not looking like that of the Figma design, please try to use the proper colors and with on all input forms and Textarea field.
- Please work on the social media icon on your design to be the same as the Figma design and also do not forget to add the dark dash on the footer section.

##### Your design 
![Screenshot from 2022-07-05 19-17-30](https://user-images.githubusercontent.com/58771507/177392114-db12504f-c4b1-4324-81da-2f78e6cb135b.png)

##### Figma design 

![Screenshot from 2022-07-05 19-31-56](https://user-images.githubusercontent.com/58771507/177392878-2f34059b-5282-45d1-81a5-3ec5fa54253e.png)